contracts/coordinator/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+pub mod client;
```

Comment:
I suggest to reexport the client instead to prevent `coordinator::client::Client` calls